backend/models.py

```diff
@@ -378,13 +377,13 @@ def get_subtotal(self) -> Decimal:
             subtotal += item.get_total_price()
         return Decimal(round(subtotal, 2))
 
-    def get_tax(self, amount: Decimal = 0.00) -> Decimal:
-        amount = amount or self.get_subtotal()
+    def get_tax(self, amount: Decimal = Decimal(0.00)) -> Decimal:
+        amount = amount or Decimal(self.get_subtotal())
```

Comment:
Decimal is already a decimal isn't it?

Reply:
In this case it's still was considered by python and mypy as a `float` that's why I used `Decimal` directly on it.

Reply:
To create Decimal it always have to be called first with a given number as a argument.

Reply:
It have been fixed.